.travis.yml

```diff
@@ -26,3 +26,14 @@ env:
   global:
   - secure: Kf44XQFpq2QGe3rn98Dsf5Uz3WXzPDralS54co7sqT5oQGs1mYLYZRYz+I75ZSo5ffZ86H7M+AI9YFofqGwAjBixBbqf1tGkUh3oZp2fN3QfqzazGV3HzC+o41zALG5FL+UBaURev9ChQ5fYeTtFB7YAzejHz4y5E97awk934Rg=
   - secure: QbNAu0jCaKrwjJi7KZtYEBA/pYbTJ91Y1x/eLAJpsamswVOvwnThA/TLYuux+oiZQCiDUpBzP3oxksIrEEUAhl0lMtqRFY3MrcUr+si9NIjX8hmoFwkvZ5o1b7pmLF6Vz3rQeP/EWMLcljLzEwsrRXeK0Ei2E4vFpsg8yz1YXJg=
+cache:
+  directories:
+  - $HOME/.sbt/0.13
```

Comment:
Anything in particular you're looking to cache here?

My stance is that you should cache as little as possible to avoid false positives, and very carefully start opting into what you want to cache from one CI run to another.

For instance here there are (what I think are) dependency graph results under `~/.sbt/0.13/dependency/` which I'm not sure you want to uncache and recache every run.